### PR TITLE
Update README with information about JS.

### PR DESCRIPTION
**Pull request by pikinier20**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,18 +23,31 @@ If you see any bugs, please inform us about it by creating issue in our reposito
 
 ### Basics
 
-Inkuire consists of three modules:
+Inkuire consists of four modules:
```
Comment (KacperFKorban) [inline README.md:26]: Technically it's like nine or sth like that now, right?

File: README.md [hunk 2]
```diff
@@ -23,18 +23,31 @@ If you see any bugs, please inform us about it by creating issue in our reposito
...
-3. Inkuire IntelliJ Plugin - This module allows you to run queries from IntelliJ.
+2. Inkuire HTTP Engine - This module is responsible for executing queries using REST API
+3. Inkuire JS Engine - This module is responsible for embedding search engine in Dokka documentation.
+4. Inkuire IntelliJ Plugin - This module allows you to run queries from IntelliJ.
 
 ---
 ### Input signatures
 
-Currently tool accepts any correct Kotlin signature.
+Currently tool accepts any correct Kotlin signature. Additionally, it supports `*` character as wildcard, which matches any type
```
Comment (KacperFKorban) [inline README.md:35]: The addition is only a "top level" star projection

File: README.md [hunk 3]
```diff
@@ -43,17 +56,54 @@ In order to generate database, you need to configure Dokka in your project and a
 Running dokka task will generate .json files containing information further needed by engine.
 Make sure you configure proper version of Dokka.
 
-See example: [Kotlin Standard Library](https://github.com/BarkingBad/kotlin-dokka-stdlib/blob/inkuire/build.gradle)
+Depending on whether you want to run HTTP instance or embed search engine inside Dokka, you can use two different Dokka plugins.
+In both cases you need to add to your `build.gradle` repository containing plugin JAR: it can be local Maven repository or Bintray.
+
+EngineJS:
+If you want to generate Dokka documentation with embedded Inkuire search engine, add this line to `build.gradle` dependencies block:
+```Groovy
```
Comment (BarkingBad) [inline README.md:64]: Groovy uses single quotes for strings. However, I would opt for Kotlin syntax anyway

**Review by KacperFKorban — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -43,17 +56,54 @@ In order to generate database, you need to configure Dokka in your project and a
...
+  "port": { "port": 8080 },
+  "dbPaths": [
+    {"path" : "path to functions db files relative to <root>/scripts"}
+  ],
+  "ancestryGraphPaths": [
+    {"path" : "path to ancestry graph db files relative to <root>/scripts"}
+  ]
+}
+```
+First two options are mandatory, but have no influence on engine runtime.
```
Comment (BarkingBad) [inline README.md:104]: I would like to point out here why we do not link generate config automatically (becuase of weird source set names and weird dependencies and Inkuire yet not support multiplatform thus user has to define his preferred source sets to be loaded)

File: README.md (hunk 1 above)
Comment (BarkingBad) [reply README.md:26]: I guess, from user perspective we should not bring up irrelevant modules

**Review by BarkingBad — CHANGES_REQUESTED** [review]
Mostly, I would apprecieate to add 
```
repositories {
	maven {
		url  "https://dl.bintray.com/virtuslab/Inkuire"
	}
}
```

to the installation guide, to make users familiar with our repo

**Review by BarkingBad — APPROVED** [review]

**Pull request merged**